app/services/automation_rules/action_service.rb

```diff
@@ -21,6 +21,14 @@ def perform
 
   private
 
+  def send_attachments(_file_params)
+    return if @rule.event_name == 'message_created'
+
+    params = { content: nil, private: false, attachments: [@rule.file_blob] }
+    mb = Messages::MessageBuilder.new(@administrator, @conversation, params)
```

Comment:
where do get the value of `@administrator` from?

Reply:
From here: 
https://github.com/chatwoot/chatwoot/pull/4266/files#diff-f127d9ac5d9df6d9a8903e5929efde7a571cec4c374ff2f13dc61ce68ca8b2c2R91

Reply:
*https://github.com/chatwoot/chatwoot/blob/971755b84570e8c23f83193dd98fec0f93482356/app/services/automation_rules/action_service.rb#L82